src/mixins/filing-mixin.ts

```diff
@@ -103,7 +104,7 @@ export default class FilingMixin extends DateMixin {
       },
       correction: {
         comment: '',
-        correctedFilingDate: correctedFiling.submittedDate,
+        correctedFilingDate: this.dateToYyyyMmDd(submittedDate),
```

Comment:
First, Vysakh confirmed that the Legal API does nothing with this value. We probably save it as part of the corrected filing's data for better traceability only.

Second, here are our time formats and their meanings:
https://github.com/bcgov/bcrs-shared-components/blob/main/src/interfaces/date-interfaces.ts

Additionally, we sometimes use `new Date('2023-06-15 12:34:56 GMT')`. Note that this is different from `new Date('2023-06-15 12:34:56 PDT')`. We use this format because the Date constructor understands this date-time format consistently. Other formats may or may not work depending on Node/browser/locale.

[ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)

Just to make date-times more complicated, in addition to timezone offset, we also have Pacific Daylight Time (PDT) vs Pacific Standard Time (PST).

Reply:
To actually answer your question, if we save 2023-06-15 then the Legal API should understand this to mean "June 15, 2023 Pacific".

If the API needs a time then it should append 00:00:00.

Example 1: "2023-06-15T00:00:00 PDT"
Example 2: "2023-06-15T07:00:00 GMT"

Reply:
The same thing should apply here.

If you're seeing `2023-06-15:T00:00:00+00:00` in the database then the API did not save the date as a correct UTC/GMT value.

If you save `2023-06-15` then the API should return a value that means "June 15 Pacific". It is not up to the UI to convert `2023-06-15:T00:00:00+00:00` to "June 15 Pacific".

Reply:
The same thing should apply here.

If you're seeing `2023-06-15:T00:00:00+00:00` in the database (or API response) then the API is in error.

If you save `2023-06-15` then the API should return a value that means "June 15 Pacific". It is not up to the UI to convert `2023-06-15:T00:00:00+00:00` to "June 15 Pacific".

Reply:
If the API insists on returning a date-time (instead of just a date), then it should look like `2023-06-15:T70:00:00+00:00` (note the 7h offset to convert from GMT to Pacific).

Reply:
OK, noted Sev. Thanks for clearing that up!

Reply:
One more thing.

For some fields, the Legal API actually does return YYYY-MM-DD, which is understood to be Pacific. (No time.)